docs/src/pages/[platform]/components/heading/demo.tsx

```diff
@@ -1,21 +1,41 @@
 import * as React from 'react';
-import { Heading, HeadingLevel } from '@aws-amplify/ui-react';
+import { Heading, HeadingLevel, HeadingProps } from '@aws-amplify/ui-react';
 
 import { HeadingPropControls } from './HeadingPropControls';
 import { Demo } from '@/components/Demo';
 
-const propsToCode = (level: HeadingLevel) => {
-  return `<Heading level={${level}}>Heading</Heading>`;
+const propsToCode = (
+  level: HeadingLevel,
+  isTruncated: boolean,
+  value: string
+) => {
+  return `<Heading
+  width='30vw'
+  level={${level}} ${isTruncated ? `\n  isTruncated={true}>` : ''}
```

Comment:
The closing `>` for `Heading` should go on its own line, not after `isTruncated={true}>`

Reply:
done